### DISCUSSION TITLE
[Feature Request] Organisation Chats

### DISCUSSION THREAD

**Discussion by FAXES** [community/community#49462] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

A system where we can have a chat system right in GitHub in organisations would be unique. This could save us from using an external service like Teams, Meet, Signal, etc to chat with each other.

Discussions do exist yes, however, they are more useful for threads of topic-based conversations, not general team chats.

This would be something cool to see in GitHub.

**Comment by Vibezz07** [comment]
IM FAXES #1 FAN
